src/SentrySdk.php

```diff
@@ -0,0 +1,287 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sentry;
+
+use Sentry\Integration\IntegrationInterface;
+use Sentry\State\Hub;
+use Sentry\State\HubInterface;
+use Sentry\State\Scope;
+
+/**
+ * This class is the main entry point for all the most common SDK features.
+ *
+ * @author Stefano Arlandini <sarlandini@alice.it>
+ */
+final class SentrySdk
+{
+    /**
+     * @var HubInterface|null The current hub
+     */
+    private static $currentHub;
+
+    /**
+     * Constructor.
+     */
+    private function __construct()
+    {
+    }
+
+    /**
+     * Initializes the SDK by creating a new hub instance each time this method
+     * gets called.
+     *
+     * @return HubInterface
+     */
+    public static function init(): HubInterface
+    {
+        self::$currentHub = new Hub();
+
+        return self::$currentHub;
+    }
+
+    /**
+     * Gets the current hub. If it's not initialized then creates a new instance
+     * and sets it as current hub.
+     *
+     * @return HubInterface
+     *
+     * @deprecated
```

Comment:
don't really see the point of adding yet a new method that will be deprecated anyways...

Reply:
The reason is that this method is needed to make the (also) deprecated method `Hub::getCurrentHub` work